.gitignore

```diff
@@ -4,6 +4,9 @@
 !.vscode/launch.json
 *.code-workspace
 
+# PyCharm
+.idea/
```

Comment:
ideally this kind of line should be done on your general .gitignore, not the project's gitignore (but honestly it's not a big deal). Because the project itself doesn't generate any .idea folder, and we don't want to clutter the .gitignore with all the editor specific things that might be needed.

Reply:
I added it since I saw that there are vscode exclusions just above. I removed the change to the .gitignore file and added to my local exclusions file instead.